### ISSUE TITLE
Problem with  EntropyBottleneck

### ISSUE THREAD

**Issue by duding0521** [tensorflow/compression#11]
Hello,
I'm trying to use some of your layers ,but I find some problems.When I read the  bls.py ,I find  you used an auxiliary loss，and the code is  'aux_step = aux_optimizer.minimize(entropy_bottleneck.losses[0])' , but I couldn't find losses in the layer of EntropyBottleneck. Can you help with this?

**Comment by jonycgn** [comment]
Hi,

I'm not sure I understand what you're trying to do. The auxiliary loss is used for estimating the quantiles of the entropy model (the density model used for estimating cross entropy). This is necessary so that the number of symbols of the range coder can be determined. The loss will be created when the layer is built (typically, that happens when it is called on the first input).

I'm closing this issue, because I would like you to post any usage questions to our [Google group](https://groups.google.com/forum/#!forum/tensorflow-compression). If you think there is a problem in the library, feel free to reopen it and provide more details.

Thanks!
